tests/robustness/traffic/traffic.go

```diff
@@ -181,5 +197,33 @@ func (p Profile) WithoutCompaction() Profile {
 type Traffic interface {
 	Run(ctx context.Context, c *client.RecordingClient, qpsLimiter *rate.Limiter, ids identity.Provider, lm identity.LeaseIDStorage, nonUniqueWriteLimiter ConcurrencyLimiter, finish <-chan struct{})
 	ExpectUniqueRevision() bool
-	WithoutCompact() Traffic
+}
+
+func RunCompactLoop(ctx context.Context, c *client.RecordingClient, period time.Duration, finish <-chan struct{}) {
+	var lastRev int64 = 2
```

Comment:
Why is `lastRev` always set to 2 here?

Reply:
It's initial value.

Reply:
Ahh there is only one possible compaction loop per simulation. I understand now thanks.